images/leap/jeos/15.2/image.yaml

```diff
@@ -0,0 +1,9 @@
+include-paths:
+  - base/jeos/leap15
```

Comment:
`include-paths` is intended for the version suffix not the whole thing. It should read (analogue to sles):

```yaml
include-paths:
  - leap15/1
  - leap15/2
```